docs/advanced_reference/commands/autonomy_service.md

```diff
@@ -11,15 +11,46 @@ This command group consists of a number of functionalities to manage [the life c
 `--dry-run`
 : Perform a dry run for the transaction.
 
-`--use-ethereum`
-: Use the Ethereum chain profile to interact with the Autonolas Protocol registry contracts. This option requires that you define the following environment variable:
+`--use-celo`                      
+:   Use the `Celo` chain profile to interact with the Autonolas Protocol registry contracts.
```

Comment:
```suggestion
: Use the Celo chain profile to interact with the Autonolas Protocol registry contracts.
```
Please apply these changes on the rest of entries (leave only 1 space after :   and remove code quotes on chain names)